src/commands/set.js

```diff
@@ -116,27 +119,28 @@ const setNethash = (dotNotation, value) => {
 			throw new ValidationError(NETHASH_ERROR_MESSAGE);
 		}
 	}
-	return setString(dotNotation, value);
+	return setValue(dotNotation, value);
 };
 
 const handlers = {
-	'api.node': setString,
+	'api.nodes': setArrayString,
 	'api.network': setNethash,
 	json: setBoolean,
-	name: setString,
+	name: setValue,
```

Comment:
Should I be able to set the name to an empty string by running `set name` with no inputs?

Reply:
Depends on usecase, but would the empty name be problem?

Reply:
Mostly in `src/index.js`.

Reply:
so it uses the default value `lisky` and it seems to be logical

Reply:
I guess so, but you still get a weird looking message as in the empty array case.